packages/cli/src/commands/repo/lint.ts

```diff
@@ -233,9 +233,11 @@ export async function command(opts: OptionValues, cmd: Command): Promise<void> {
     if (runFailed) {
       console.log(chalk.red(`Lint failed in ${relativeDir}`));
       failed = true;
+    } else if (sha) {
+      outputSuccessCache.push(sha);
+    }
 
-      // When doing repo lint, only list the results if the lint failed to avoid a log
-      // dump of all warnings that might be irrelevant
+    if (failed || !opts.quiet) {
```

Comment:
shouldn't this be: 🤔 

```suggestion
    if (failed && !opts.quiet) {
```

Reply:
Logic here is that `failed` will be logged out to console no matter what `opts.quiet` value is. But warnings will not be logged to console if `opts.quiet` is `true`.